src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -1697,7 +1697,11 @@ NOTE: if you're sure you want to do this, please open an issue as to why. In the
         builder.ensure(TestHelpers { target: compiler.host });
```

Comment:
Remark: the compiletest `Step::run` is some gigantic convoluted build step, lol. I might revisit that in the future and see if I can try to make it uhhh less insane.